### PR TITLE
Kevin/mailman popup

### PR DESCRIPTION
**Pull request by klpittman**

### CODE REVIEW

File: myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html [hunk 1]
```diff
@@ -34,19 +32,31 @@
...
-                        <div>
-                            <label>Select all</label> <input type="checkbox"/>
+                        <div class="clearfix">
+                            <div class="pull-right">
+                                <label for="select-all-sections">Select all</label> <input id="select-all-sections" type="checkbox"/>
+                            </div>
                         </div>
-                        <ul>
+                        <ul class="clearfix unstyled-list">
                             {{#each secondary_lists}}
```
Comment (klpittman) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: I found this part a little confusing. `{{#each secondary_lists}}`  makes it sound like we're looping over each email list, but in my head the intent here is to loop over each **section**.

File: myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html [hunk 2]
```diff
@@ -34,19 +32,31 @@
...
-                            <li class="secondary_single">
+                            <li class="secondary_single clearfix">
                                 {{#unless list_exists}}
-                                <label for="secondary_single_{{ list_address }}">
-                                    {{ list_address }}@uw.edu
```
Comment (klpittman) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: `{{ list_address }}@uw.edu` I think this should be the section name. Let me know if I'm misunderstanding your intent though.

File: myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html (hunk 1 above)
Comment (klpittman) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: I found this part a little confusing. `{{#each secondary_lists}}`  makes it sound like we're looping over each email list, but in my head the intent here is to loop over each **section**.

Comment (fanglinfang) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: click on Create email list with billsea,  does this make sense to you now?

File: myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html (hunk 2 above)
Comment (fanglinfang) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: Added section name

File: myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html (hunk 1 above)
Comment (klpittman) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: Yes, I was able to see the list. Can you describe what `{{#each secondary_lists}}` is supposed to do though? Maybe that will clarify it for me.

File: myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html [hunk 3]
```diff
@@ -3,25 +3,23 @@
...
     <div class="modal-content">
-        <div class="modal-header">
-            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <form id="emaillist{{course_abbr}}{{course_number}}{{section_id}}" method="POST" action="{% url 'myuw_emaillist_api' %}">{% csrf_token %}
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
 
-            <h4>Create Mailing List</h4>
-        </div>
+                <h4>Create Mailing List</h4>
```
Comment (fanglinfang) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: Is this a tab?  we use spaces for indentation.

Comment (klpittman) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: Let me try to fix

File: myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html (hunk 1 above)
Comment (fanglinfang) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: we allow the primary instructor to create single course lists for secondary sections, right?

File: myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html (hunk 3 above)
Comment (fanglinfang) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: if it is space, no change needed

Comment (klpittman) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: Looks like they are spaces. I adjusted the indenting though.

File: myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html (hunk 1 above)
Comment (klpittman) [inline myuw/templates/handlebars/card/instructor_schedule/mailman/request_email_lists.html]: > we allow the primary instructor to create single course lists for secondary sections, right?

Yes, they should be able to.

**Pull request merged**